api/swagger/server/restapi/configure_massastation.go

```diff
@@ -68,6 +68,10 @@ func configureMassaStationAPI(api *operations.MassastationAPI, config config.App
 	api.ServerShutdown = func() {
 		close(shutdown)
 	}
+	api.Logger = func(msg string, args ...interface{}) {
+		// Call the appropriate method of the *zap.Logger object
+		MSConfig.Logger.Infof(msg, args...)
```

Comment:
the comment is a TODO? you always log with info level, are you sure it's always this level?

Reply:
I removed it, it was just a description of the function